lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -52,16 +65,19 @@ const rule = (primary) => {
 
 			parseMediaQuery(atRule).forEach((mediaQuery) => {
 				if (isMediaQueryInvalid(mediaQuery)) {
-					// Queries that fail to parse are invalid.
-					complain(atRule, atRuleParamIndexValue, mediaQuery);
+					if (!shouldIgnoreNode(mediaQuery, secondaryOptions)) {
+						// Queries that fail to parse are invalid.
+						complain(atRule, atRuleParamIndexValue, mediaQuery);
+					}
 
 					return;
 				}
 
 				mediaQuery.walk(({ node, parent }) => {
-					// All general enclosed nodes are invalid.
```

Comment:
[question] Is this comment removal intentional? It seems still necessary to me... 🤔